lib/datastore/transaction.js

```diff
@@ -42,6 +42,11 @@ var extend = require('extend');
  *     connection to Google Cloud Datastore.
  * @param {string} projectId - Dataset ID. This is your project ID from the
  *     Google Developers Console.
+ *
+ * @example
+ * var Transaction = require('gcloud/lib/datastore/transaction');
```

Comment:
This is added to the developer documentation, which is hidden from generated docs (as far as I know), however because the VM runs this with the other examples in the file, we now have a proper `transaction` variable that can be used for all the examples that follow.